setup-middleware.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const bodyParser = require('body-parser').json({ limit: '50mb' });
+const path = require('path');
+const date = require('date-and-time');
+const { ensureDirSync, writeJsonSync } = require('fs-extra');
+
+function reportViolations(req, res, options) {
+  const REPORT_TIMESTAMP = date.format(new Date(), 'YYYY-MM-DD-HH_mm_ss');
+  let outputDir = path.join(options.root, 'ember-a11y-report');
+  let outputPath = path.resolve(
+    path.join(outputDir, `${REPORT_TIMESTAMP}.json`)
+  );
+
+  ensureDirSync(outputDir);
+  writeJsonSync(outputPath, req.body);
```

Comment:
I _think_ these two calls are redundant, right? Doesn't `writeJsonSync` internally `ensureDirSync`?

Reply:
It does not.

Reply:
Whoops! Sorry, I was thinking of `outputJsonSync` 

https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson-sync.md